sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidSqlValidator.java

```diff
@@ -796,6 +797,20 @@ public void validateCall(SqlCall call, SqlValidatorScope scope)
     super.validateCall(call, scope);
   }
 
+  @Override
+  protected void validateWindowClause(SqlSelect select)
+  {
+    SqlNodeList windows = select.getWindowList();
+    for (SqlNode sqlNode : windows) {
+      if (SqlUtil.containsAgg(sqlNode)) {
+        throw DruidException.forPersona(DruidException.Persona.USER).ofCategory(DruidException.Category.UNSUPPORTED)
```

Comment:
in this class all exceptions should be emitted with a link to the context(`sqlNode`) they appear

Reply:
updated